ui/app/ducks/gas/gas.duck.js

```diff
@@ -192,17 +177,6 @@ export function setCustomGasLimit(newLimit) {
   }
 }
 
-export function setBasicPriceEstimatesLastRetrieved(retrievalTime) {
-  return {
-    type: SET_BASIC_PRICE_ESTIMATES_LAST_RETRIEVED,
-    value: retrievalTime,
-  }
-}
-
-export function resetCustomGasState() {
```

Comment:
I was about to ask why this was removed as well, but I think I see now. It's dead code.